sql/src/main/java/io/crate/execution/dsl/phases/FileUriCollectPhase.java

```diff
@@ -105,6 +114,7 @@ public FileUriCollectPhase(StreamInput in) throws IOException {
         }
         this.executionNodes = nodes;
         toCollect = Symbols.listFromStream(in);
+        inputFormat = WriterProjection.InputFormat.values()[in.readInt()];
```

Comment:
Minor: You could replace this with `in.readVInt()` since this will safe some space for low range integers (we have only 0-1 here).